### PR TITLE
fix for asset_path on production

### PR DESCRIPTION
**Pull request by itai-sagi**

### CODE REVIEW

File: vendor/assets/javascripts/react_rails_img.js.jsx.erb [hunk 1]
```diff
@@ -4,15 +4,10 @@ prefix = Rails.application.config.assets.prefix
 should_compute_assets_path = Rails.env.production? # for performance
 if should_compute_assets_path
   images_dir = Rails.root.join('app/assets/images/').to_s
-  assets.each_file do |file_path|
+  Dir["#{images_dir}/**/*"].each do |file_path|
```
Comment (rainchen) [inline vendor/assets/javascripts/react_rails_img.js.jsx.erb:7]: using `Dir["#{images_dir}/**/*"]` will not be allowed to use assets placed in vendor gems.

I think a better solution is using following way:

```
images_enumerator = defined?(assets) ? assets.each_file : Dir["#{images_dir}**/*.*"]
images_enumerator.each do |file_path|
  file_name = file_path.sub(images_dir, '')
  file_path = asset_path(file_name)
end
```

Comment (rainchen) [inline vendor/assets/javascripts/react_rails_img.js.jsx.erb:7]: @itaitay I would like to accept your PR if you can update the codes as I posted.

Comment (itaitay) [inline vendor/assets/javascripts/react_rails_img.js.jsx.erb:7]: I would today :)

Comment (rainchen) [inline vendor/assets/javascripts/react_rails_img.js.jsx.erb:7]: @itaitay how is it going? looking forward to see the changes

**Comment by rainchen** [comment]
fixed in this PR https://github.com/rainchen/react-rails-img/pull/3

**Pull request closed without merging**